src/cls/rgw/cls_rgw_types.h

```diff
@@ -612,6 +615,24 @@ enum cls_rgw_reshard_status {
   CLS_RGW_RESHARD_DONE        = 2,
 };
 
+static inline std::string to_string(const enum cls_rgw_reshard_status status)
```

Comment:
i like this very much, because it acts as an overload of [std::to_string()](https://en.cppreference.com/w/cpp/string/basic_string/to_string) (which is defined only for integral types) :+1: